### ISSUE TITLE
Broken with pari-2.13.0: Undefined symbol "isirreducible"

### ISSUE THREAD

**Issue by yurivict** [sagemath/cypari2#98]
```
Traceback (most recent call last):
  File "x.py", line 1, in <module>
    import cypari2
  File "/usr/local/lib/python3.7/site-packages/cypari2/__init__.py", line 1, in <module>
    from .pari_instance import Pari
  File "cypari2/pari_instance.pyx", line 1, in init cypari2.pari_instance
ImportError: /usr/local/lib/python3.7/site-packages/cypari2/gen.so: Undefined symbol "isirreducible"
```

cypari2-2.1.1

FreeBSD 12.2

**Comment by yurivict** [comment]
The function has been renamed:

> COMPAT|19| - rename libpari function isirreducible -> polisirreducible

and it was deprecated in 2.13.0.

**Comment by antonio-rojas** [comment]
This is already fixed with 1a4be24489c2e457727d76bf6dd71daa5c45aee4 (but not yet released)

**Comment by videlec** [comment]
Indeed
